src/cmd.ts

```diff
@@ -137,11 +137,17 @@ function createCommandWithSharedOptions(name, description) {
 			"Include single-episode torrents in the search",
 			fallback(fileConfig.includeEpisodes, false)
 		)
+		.option(
+			"--include-season-pack-episodes",
+			"Include episodes from a season pack in the search",
+			fallback(fileConfig.includeSeasonPackEpisodes, false)
```

Comment:
this will change the behavior for everyone when we release the new version, so we have a couple options:

- bump the major version and tell everyone about breaking changes (more work)
- default to `true` (tech debt)
- wait until another breaking change comes along and bundle it with that
- offer some other sort of escape hatch so that we don't need another argument here

Reply:
I did think about the potential to rely on includeEpisodes if undefined, for instance if they are using a deprecated version of the template/config or something.

I'm not sure how it would best be handled, but if we have includeEpisodes = true and an undefined (neither true or false) includeSeasonPackEpisodes,default to previous behavior (includeSeasonPackEpisodes = true)

We obviously don't want to assume anything about preferences, but changing behavior is sort of a concern i thought about, especially with people rarely touching their configurations after initial setup.